src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -158,15 +146,15 @@
                [:<>
                 [:> Replay]
                 [:span "Re-connect with remote"]]])
-            [button {:on-click #(swap! merge-open? not)
+            #_ [button {:on-click #(swap! merge-open? not)
```

Comment:
@shanberg

Reply:
@tangjeff0 This had been moved into the DB menu since it affects the DB directly and is mostly a one-time action, but the rest of the code hasn't caught up to that change yet. I'll uncomment this for now.